jenkins/job-configs/kubernetes-jenkins-pull/charts-pull.yaml

```diff
@@ -0,0 +1,146 @@
+- job-template:
+    name: '{git-project}-pull-{suffix}'
+    # defaults
+    success-comment: ''
```

Comment:
You can just substitute the values here into the appropriate locations, no need to variables.